src/libs/NetworkConnection.ts

```diff
@@ -72,42 +73,51 @@ Onyx.connect({
  * internet connectivity or not. This is more reliable than the Pusher
  * `disconnected` event which takes about 10-15 seconds to emit.
  */
-function subscribeToNetInfo(): void {
-    // Note: We are disabling the configuration for NetInfo when using the local web API since requests can get stuck in a 'Pending' state and are not reliable indicators for "offline".
+function subscribeToNetInfo() {
```

Comment:
The name and the comment got somewhat outdated.

New name suggestion: `subscribeToNetworkStatus`

New comment suggestion:

> Monitor internet connectivity and perform periodic backend reachability checks

Reply:
(I suggested moving the details about NetInfo to a [local comment](https://github.com/Expensify/App/pull/38377#discussion_r1530075340))